javascript/statements.json

```diff
@@ -1008,11 +1008,11 @@
               "version_added": "12"
             },
             "firefox": {
-              "version_added": "13",
+              "version_added": "14",
```

Comment:
Firefox 13 for developers still has it listed! (And not Firefox 14 for developers). Please change both pages if this change was indeed correct

I wasn't able to find the bug to decide if 13 or 14 is correct.

Reply:
13 is correct, I wanted to add 14 for Android as 13 doesn't exist for Android but accidentally also changed the Firefox desktop version. Good catch!